thoth/license_solver/output_creator.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# license-solver
+# Copyright(C) 2021 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Class create output dictionary."""
+
+import json
+import sys
+import logging
+from .package import Package
+from typing import Dict, Any
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class OutputCreator:
+    """Propose of this class is to create dictionary for all packages (input)."""
+
+    file: Dict[Any, Any] = dict()
+
+    def add_package(self, package: Package, warning: bool = False) -> None:
+        """
+        Add package to dictionary.
+
+        :param package: Package data
+        :param warning: default False, if true create warning in package info
+        :return: None
+        """
+        package_data: Dict[str, Any] = {
+            "license": package.license,
+            "license_version": str(package.license_version),
+            "classifier": package.classifier,
+        }
+
+        if warning:
+            package_data["warning"] = str(True)
+        else:
+            package_data["warning"] = str(False)
```

Comment:
True/False are serializable into JSON or is there any other reason for this casting?